### ISSUE TITLE
Sickrage fails to open after install, can't find localhost

### ISSUE THREAD

**Issue by boyeah2000** [SickRage/SickRage#2081]
### Before submitting your issue:

Enable debug logging in SickRage settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit:
OS: Win10
What you did: installed, fails to open
What happened: opened sickrage, browser says can't find localhost. I let it originally try 8081 then uninstalled and tried port 8080. Forgive my ignorance, but I'm not sure what i'm doing wrong or if those ports are being used. I tried finding answers in the forum but failed for two reasons 1) I can't find the config file to edit to try different ports and 2) I wouldn't know which port to use as an alternative. Any help is greatly appreciated!
What you expected:
Logs:
```
PASTE LOGS HERE
```

**Comment by SickRage2** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Include basic information: Branch/Commit, OS, What you did, What happened, What you expected
- [ ] Enable debug logging (be sure to disable after the bug is fixed)
- [ ] Post debug logs, either inline (for smaller logs) or using gist

Please make sure you also read [how to create an issue](https://github.com/SickRage/sickrage-issues/blob/master/README.md) and followed all of the steps. 

The title should describe your issue. Having "SR not working" or "I get this bug" for 100 issues, isn't really helpful. We will close issues if there isn't enough information.

Sometimes the devs may seem like grunts and respond with short answers. This isn't (always) because the dev hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let us know, and this bot may get updated to automatically answer you.

Thanks!

**Comment by Shah-Parth** [comment]
Just out of curiosity, have you tried using `127.0.0.1:8080`?

**Comment by rrik** [comment]
I'm having the same issue, installing with the bundled installer for Win 10. I tried installing on a different port, and logging in from 127.0.0.1:8081, but so far they aren't working. Re-booting and starting/stopping the service does not work.

**Comment by neoatomic** [comment]
Probably Windows firewall blocking.?

If i recall right you can start Sickrage from the command prompt, at that point a windows firewall popup will appear that you can approve.

If all fail than checkout the Wiki guides to install sickrage manually on Windows. (without the installer) : 

https://github.com/SickRage/SickRage/wiki/SickRage-Windows-Installer#manual-installation-guides-for-windows

**Comment by neoatomic** [comment]
Just did a fresh install on win 10 pro x64, and all is working fine.

Using `localhost:8081`

So cant explain it, unless you are using a very old sickrage installer. v0.5 is the latest. Check the wiki for the link.

![1](https://cloud.githubusercontent.com/assets/7928052/17080198/9528b708-5126-11e6-8e37-ac1b4aea5f34.png)

**Comment by NKrukov** [comment]
Not sure why this was closed until the original poster has had his issue resolved.  I'm getting the same issue on Windows 10 64 Bit.  Used the SickRageInstaller.exe I downloaded from the Sick Rage website.  I try to open it via both 127.0.0.1:8081, and localhost:8081 with no joy.  Currently troubleshooting this issue on my own, would have been nice to have some troubleshooting options on this thread.

TLDR:  Don't close an issue until its actually resolved.

**Comment by TMac7373** [comment]
Can you be a bit more specific or maybe link me to the forum post?  I'm having the same problem.  When you say you changed the port, is that just in the address bar or is it in a setting somewhere?

**Comment by skeeve42** [comment]
I have the same problem on a mostly fresh installation of Windows 10 (I used the 'Reset this PC' option within Windows 10). Using the SickRage Windows installer, it says it completes and the service is running but I am unable to access SickRage on the default port (8081). 
I uninstalled and then reinstalled using a different port (8090) and I can access it fine.

**Comment by NKrukov** [comment]
The way it fixed for me;  When you are running the installation, when you get to the part where you set the port during installation [https://cloud.githubusercontent.com/assets/7928052/9738293/15725f7c-564b-11e5-997b-edf09e8f50bf.png](Here) you set the port as something other than 8081.  I set mine to 8082, and it worked.  When installation is done, go to http://localhost:8082, and it'll work!

Hope that helps!

**Comment by TMac7373** [comment]
@skeeve42  @NKrukov that did it.  thanks so much for your help!

**Comment by ccie9360** [comment]
Add me to this list, does not work on default port 8081. Yes I tried the loopback 127... Yes I validated windows firewall was not blocking. 

Change default port and it worked great, wonder why its still doing this....
